codec.go

```diff
@@ -403,7 +409,7 @@ func (cdc *Codec) checkConflictsInPrio_nolock(iinfo *TypeInfo) {
 				}
 			}
 			if !inPrio {
-				panic(fmt.Sprintf("%v conflicts with %v other(s). Add it to the priority list for %v.", cinfo.Type, iinfo.Type))
+				panic(fmt.Sprintf("%v conflicts with %v other(s). Add it to the priority list", cinfo.Type, iinfo.Type))
```

Comment:
The message is wrong.  "%v other(s)" was supposed to count the number of conflicts... which we could do if `var inPrio int` and we increment.

Reply:
Yeah, I filed https://github.com/tendermint/go-wire/issues/78 delegating that you :) as vet on Go1.10 won't compile.